package.json

```diff
@@ -5,14 +5,18 @@
   "main": "script.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build:sass": "gulp sass"
+    "build:sass": "gulp sass",
+    "build:ts": "gulp ts"
```

Comment:
Add
- independent script for build ts 
- independent script for build scss
- independent script for watching ts and scss changes